main.go

```diff
@@ -56,8 +62,17 @@ var (
 	mutex        sync.Mutex
 	configFile   = flag.String("c", "conf.json", "config file location")
 	generateKey  = flag.Bool("g", false, "generate an API key")
+	generateSigningKey  = flag.Bool("k", false, "Generate a signing key pair")
+	keyName      = flag.String("kn", "", "Name for the siging key")
+	keyEmail     = flag.String("ke", "", "Email address")
 	parsedconfig = conf{}
 	mywatcher    *fsnotify.Watcher
+
+
+	//Create a package level time function so we can mock it out
+	Now = func() time.Time {
+		return time.Now()
+	}
```

Comment:
Curious about this guy, why not just use the `time` package? It appears in the tests the only thing that changes is that a set value is returned.

Reply:
That's the reason - the current time is written to the Release file so I can't compare it to a known good value without fixing the time, so wrapping time.Now() in an anonymous function allows it to be overridden.

I found this snipped on Stack Overflow, and it seemed a lot easier than importing a full time mocking library.